pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -579,7 +592,7 @@ def _handle_image_build_failures(self):
 
         failed_images = list(failed_map.keys())
         run_details.update_status('UNSTABLE')
-        run_details.update_description(f'Failed images: f{", ".join(failed_images)}<br/>')
+        run_details.update_description(f'Failed images: {", ".join(failed_images)}<br/>')
```

Comment:
maybe same here only need a number, if mass build all failed the failed_image list will also fill the description

Reply:
Done in https://github.com/openshift-eng/aos-cd-jobs/pull/3791/commits/b3c52e8836c3e68cf97214fe1d7808888505b978